README.md

```diff
@@ -13,118 +13,131 @@ To find out more about MediaStore SDK, see:
+| `setRefreshToken`               |  `refeshToken :string`                                 | Customer refresh token received from login / registration / SSO endpoint                                                                                                    |
+| `setPublisher`                  | `publisherId :string`                                        | Your broadcaster ID in the Cleeng system                                                                                                                                    |
+| `setOffer`                      | `offerId :string`                                             | `offerId` is the ID of the offer created for your broadcaster in the Cleeng system                                                                                          |
+| `setCheckoutPayPalUrls`         | `{successURL: string, cancelUrl: string,errorUrl: string }` | PayPal redirection URLs, required for Paypal payment                                                                                                                        |
+| `setMyAccountPayPalUrls`        | `{successURL: string, cancelUrl: string, errorUrl: string }` | PayPal redirection URLs, required for update PayPal payment details. Query param 'message' with a readable error message will be added to errorUrl when an error will occur |
+| `setMyAccountUrl`               | `url: string`                                                 | My account URL. Needed for checkout legal notes                                                                                                                             |
+| `setOfferSelectionUrl`          | `url: string`                                                | Url to offer selection page. Recommended for CTA when the customer has no active plan                                                                                       |
+| `setTheme`                      | `styles:object`                                                      | More informations in the [Styling](#styling) section.                                                                                                                       |
+| `setVisibleAdyenPaymentMethods` | `paymentMethods: string[]`| Array of payment methods names that should be presented in Checkout and MyAccount. Available options: `applepay`, `card`, `googlepay`, `ideal`, `sofort`                                   |
+| `setHidePayPal`                 | -                                                           | Option to hide PayPal, by default PayPal will be visible when configured                                                                                                    |
+
+<b>Sample of usage</b>
```

Comment:
how about changing to 'Usage sample' so that it's consistent with similar segments below?